### PR TITLE
🐛 Fixed missing files when packaging with ninja on windows 

### PR DESCRIPTION
**Pull request by AnotherFoxGuy**
This fixes the issue with missing .dll files on the CI build

<details>
<summary>Contributor stuff</summary>

:white_check_mark: I have signed off my commits using `-s` or `Signed-off-by`\* (See: [Contributing § DCO](https://github.com/tenacityteam/tenacity/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))
:white_check_mark: I made sure the code compiles on my machine
:white_check_mark: I made sure there are no unnecessary changes in the code\*
:white_check_mark: I made sure the title of the PR reflects the core meaning of the issue you are solving\*
:white_check_mark: I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"\*

</details>

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -1351,13 +1351,16 @@ if( "${CMAKE_GENERATOR}" MATCHES "Xcode|Visual Studio*" )
...
                DESTINATION "."
                RESOURCE DESTINATION "${_APPDIR}/Resources" )
-   elseif( CMAKE_SYSTEM_NAME MATCHES "Windows" )
-      install( TARGETS ${TARGET}
-               DESTINATION "." )
+   elseif( WIN32 )
+      install( DIRECTORY "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/"
+               DESTINATION "."
+               USE_SOURCE_PERMISSIONS
+               PATTERN "*.pdb" EXCLUDE
```
Comment (Be-ing) [inline src/CMakeLists.txt:1362]: How can we get backtraces from users if we exclude debugging symbols...?

Comment (AnotherFoxGuy) [reply src/CMakeLists.txt:1362]: You need the source code to be exactly the same location to debug with VS, so adding pdbs from the CI won't give any useful backtraces

**Review by Be-ing — APPROVED** [review]
The changes look reasonable but I have not actually tested them on Windows.

**Comment by n0toose** [comment]
Is #304 relevant to this?

**Review by fossdd — APPROVED** [review]
Looks fine in my eyes.

**Pull request merged**